enhancements/network/ovs-dynamic-cpu-pinning.md

```diff
@@ -0,0 +1,116 @@
+---
+title: ovs-dynamic-cpu-pinning
+authors:
+  - "@zeeke"
+reviewers:
+  - "@trozet"
+  - "@MarSik"
+approvers:
+  - TBD
+creation-date: 2023-03-31
+last-updated: 2023-03-31
+tracking-link:
+  - https://issues.redhat.com/browse/CNF-5977
+---
+
+# OVS Dynamic CPU Pinning
+
+## Summary
+
+Allow OVS daemons to run on non reserved CPU if the network workload requires
+cycles and non-reserved CPU are idle.
+
+## Motivation
+
+OVS runs at reserved cpus when an OCP node is configured cpuManagerPolicy to `static`. (TBD doclink)
+Typically the customers want to minimize the amount of reserved cpus and leave
+as much cpus as possible available for the business related workloads.
```

Comment:
I'd say: and leave as many available CPUs as possible for business-related workloads.